plugins/modules/notification/cisco_webex.py

```diff
@@ -10,99 +10,102 @@
 
 DOCUMENTATION = '''
 ---
-module: cisco_spark
-short_description: Send a message to a Cisco Spark Room or Individual.
+module: cisco_webex
+short_description: Send a msg to a Cisco Webex Teams Room or Individual
 description:
-    - Send a message to a Cisco Spark Room or Individual with options to control the formatting.
+    - Send a msg to a Cisco Webex Teams Room or Individual with options to control the formatting.
```

Comment:
Please use correct English words.
```suggestion
    - Send a message to a Cisco Webex Teams Room or Individual with options to control the formatting.
```

Reply:
corrected